src/main/java/gov/usgs/wma/waterdata/collections/BaseController.java

```diff
@@ -1,14 +1,16 @@
 package gov.usgs.wma.waterdata.collections;
```

Comment:
Given that this is becoming a base class for the data controller, should it move up out of the collections package?